spin-up-hub/src/components/Card.vue

```diff
@@ -17,56 +17,83 @@ export default {
+              <g>
+                <path d="m1138 517.5-94-94.5c-3.3438-3.0742-5.1719-7.4609-5-12v-133c0-31.031-12.328-60.789-34.27-82.73-21.941-21.941-51.699-34.27-82.73-34.27h-133c-4.5391 0.17188-8.9258-1.6562-12-5l-94.5-94c-21.922-21.801-51.582-34.039-82.5-34.039s-60.578 12.238-82.5 34.039l-94.5 94c-3.0742 3.3438-7.4609 5.1719-12 5h-133c-31.031 0-60.789 12.328-82.73 34.27-21.941 21.941-34.27 51.699-34.27 82.73v133c0.17188 4.5391-1.6562 8.9258-5 12l-94 94.5c-21.801 21.922-34.039 51.582-34.039 82.5s12.238 60.578 34.039 82.5l94 94.5c3.3438 3.0742 5.1719 7.4609 5 12v133c0 31.031 12.328 60.789 34.27 82.73 21.941 21.941 51.699 34.27 82.73 34.27h133c4.5391-0.17188 8.9258 1.6562 12 5l94.5 94c21.793 22.047 51.5 34.453 82.5 34.453s60.707-12.406 82.5-34.453l94.5-94c3.0742-3.3438 7.4609-5.1719 12-5h133c31.031 0 60.789-12.328 82.73-34.27 21.941-21.941 34.27-51.699 34.27-82.73v-133c-0.17188-4.5391 1.6562-8.9258 5-12l94-94.5c21.801-21.922 34.039-51.582 34.039-82.5s-12.238-60.578-34.039-82.5zm-70.5 94.5-94.5 94c-21.957 22.02-34.199 51.906-34 83v133c0 4.5078-1.793 8.832-4.9805 12.02s-7.5117 4.9805-12.02 4.9805h-133c-31.094-0.19922-60.98 12.043-83 34l-94 94.5c-3.1875 3.1758-7.5 4.957-12 4.957s-8.8125-1.7812-12-4.957l-94-94.5c-22.02-21.957-51.906-34.199-83-34h-133c-4.5078 0-8.832-1.793-12.02-4.9805s-4.9805-7.5117-4.9805-12.02v-133c0.19922-31.094-12.043-60.98-34-83l-94.5-94c-3.1758-3.1875-4.957-7.5-4.957-12s1.7812-8.8125 4.957-12l94.5-94c21.957-22.02 34.199-51.906 34-83v-133c0-4.5078 1.793-8.832 4.9805-12.02s7.5117-4.9805 12.02-4.9805h133c31.094 0.19922 60.98-12.043 83-34l94-94.5c3.1875-3.1758 7.5-4.957 12-4.957s8.8125 1.7812 12 4.957l94 94.5c22.02 21.957 51.906 34.199 83 34h133c4.5078 0 8.832 1.793 12.02 4.9805s4.9805 7.5117 4.9805 12.02v133c-0.19922 31.094 12.043 60.98 34 83l94.5 94c3.1758 3.1875 4.957 7.5 4.957 12s-1.7812 8.8125-4.957 12z" />
+                <path d="m550 654.5-114.5-115-71 71 185.5 185 285.5-285-71-71z" />
+              </g>
+            </svg>
+          </span>
+          <div>
+            <span v-if="item.artifactSource" style="margin-right: 0.5rem;" class="icon"><img
+                src="/static/image/deploy-to-cloud.svg" alt="GitHub" /></span>
+            <span class="icon"><img src="/static/image/icon-github.svg" alt="GitHub" /></span>
+          </div>
+        </header>
+  
+        <article class="card-article">
+          <h3>{{ item.title }}</h3>
+          <p class="summary">
+            {{ item.summary }}
+          </p>
+          <div class="tags">
+            <span v-for="tag in item.tags" :key="tag">
+              {{ tag }}
+            </span>
+          </div>
+        </article>
+      </div>
+      </a>
     </div>
-</template>
+  </template>
 
 <style lang="scss" scoped>
+
+.card-image {
+      flex-shrink: 0;
+      width: 100%;
+      overflow: hidden;
+      padding: 0.3rem;
+      border-radius: 1.555rem;
```

Comment:
probably needs to be `1.033rem` since the outer container ("the card") is 1.333rem with a padding of 0.3rem

Reply:
i might be wrong on this as I have not gone through the entire CSS